plugin/forward/persistent_test.go

```diff
@@ -82,54 +82,6 @@ func TestCleanupByTimer(t *testing.T) {
 	tr.Yield(c4)
 }
 
-func TestPartialCleanup(t *testing.T) {
```

Comment:
do you plan to keep this test?

Reply:
no, it was already not ideal with all those sleeps